service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/PbcService.kt

```diff
@@ -203,4 +209,48 @@ class PbcService(
                 .toAny()
                 .toTxBody()
         ).throwIfFailed("Accept failed")
+
+    fun getGrant(): Grant? =
+        try {
+            grpcClientService.new().feeGrants.getFeeGrant(managerAddress, provenanceProperties.contractAddress)
+        } catch (e: StatusRuntimeException) {
+            when (e.status.code) {
+                Code.INVALID_ARGUMENT, Code.INTERNAL, Code.UNKNOWN -> null
```

Comment:
is this meant to be here? IT seems like when this returns null the endpoint will try to revoke the grant?

Reply:
Oops read the code wrong, it will skip the revoke when this is null. But the latter two errors could just be transient?